### PR TITLE
Use Revision class

### PR DESCRIPTION
**Pull request by VincentLanglet**
## Changelog

```markdown
### Fixed
- `AuditReader` compatibility with sonata-admin@4.0.0-rc.2
```

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -30,7 +30,7 @@
         "doctrine/doctrine-bundle": "^2.3",
         "doctrine/orm": "^2.8",
         "doctrine/persistence": "^2.1",
-        "sonata-project/admin-bundle": "^4.0@rc",
+        "sonata-project/admin-bundle": "dev-revision",
```
Comment (dmaicher) [inline composer.json:33]: we should probably change this?

File: composer.json [hunk 2]
```diff
@@ -47,6 +47,7 @@
         "twig/twig": "^2.10 || ^3.0"
     },
     "conflict": {
+        "sonata-project/admin-bundle": "<4.0.0-rc.1",
```
Comment (dmaicher) [inline composer.json:50]:
```suggestion
        "sonata-project/admin-bundle": "<4.0.0-rc.2",
```

?

File: composer.json (hunk 1 above)
Comment (VincentLanglet) [reply composer.json]: Oh yes, forgot ^^'

File: composer.json (hunk 2 above)
Comment (VincentLanglet) [reply composer.json]: Indeed

**Review by dmaicher — APPROVED** [review]

**Review by jordisala1991 — APPROVED** [review]

**Pull request merged**